app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -74,4 +76,10 @@ def set_access_control_headers
     headers['Access-Control-Allow-Origin'] = '*'
   end
 
+  def assign_onboarding_feature_flag(community_id:)
+    if(rand(2) == 1)
```

Comment:
This is a good solution and works, no need to change.

Another option to do this is to use `rand` without arguments. It returns float between 0..1. So in this case when we want half of the people to get the feature flag we can type:

```ruby
rand < 0.5
```

The `0.5` communicates nicely that it's 50%. If we needed to change this so that 72% of people get the flag, the change would be trivial:

```ruby
rand < 0.72
```

As said, no need to change. Just wanted to share this practice as I have found it pretty useful and clear 😀